EasyPost/Parameters/V2/Base.cs

```diff
@@ -0,0 +1,93 @@
+using EasyPost._base;
+using EasyPost.Utilities.Annotations;
+
+namespace EasyPost.Parameters.V2
+{
+    /// <summary>
+    ///     Base class for all parameters used in requests to <see cref="ApiVersion.V2"/> endpoints.
+    /// </summary>
+#pragma warning disable SA1649
```

Comment:
### Author's Note

This is a complaint that the class name does not match the file name. This is deliberate.

Reply:
I'd argue that the linter is correct and that class names should always match file names. Why can't we call this `BaseRequestParameters` or something similar so you keep your `Base` as a part of it?